src/main/telemetry/smartport.c

```diff
@@ -449,6 +448,16 @@ void handleSmartPortTelemetry(void)
                     smartPortSendPackage(id, 0);
                     smartPortHasRequest = 0;
                 }
+		else {
+		    // if no GPS send real temp
+#ifdef BARO
+		    smartPortSendPackage(id, (baroTemperature + 50)/ 100);
+#else
+		    smartPortSendPackage(id, telemTemperature1 / 10);
+#endif
+		    smartPortSendPackage(id, (baroTemperature + 50)/ 100);
```

Comment:
Indeed, that's what happens when you code before taking your morning shower ! Thx a lot for the catchup